packages/dd-trace/src/plugins/util/test.js

```diff
@@ -145,13 +175,14 @@ function getTestEnvironmentMetadata (testFramework, config) {
 
   const runtimeAndOSMetadata = getRuntimeAndOSMetadata()
 
-  const metadata = {
+  let metadata = {
     [TEST_FRAMEWORK]: testFramework,
     ...gitMetadata,
     ...ciMetadata,
     ...userProvidedGitMetadata,
     ...runtimeAndOSMetadata
   }
+  metadata = validateMetadata(metadata)
```

Comment:
I think you can simply do 
```suggestion
  const metadata = validateMetadata({
    [TEST_FRAMEWORK]: testFramework,
    ...gitMetadata,
    ...ciMetadata,
    ...userProvidedGitMetadata,
    ...runtimeAndOSMetadata
  })
```